entrypoint.sh

```
@@ -42,127 +70,129 @@ echo '::endgroup::'
-  # exit $exit_code
-# END OF fix
-else
-  echo 'ERROR: SQLFLUFF_COMMAND must be one of lint and fix'
-  exit 1
+if [[ "${SQLFLUFF_COMMAND}" == @(fix|both) ]]; then
+	echo '::group:: Running sqlfluff 🐶 ...'
+	# Allow failures now, as reviewdog handles them
+	set +Eeuo pipefail
+	# shellcheck disable=SC2086,SC2046
+	sqlfluff fix --force \
+		$(if [[ "x${SQLFLUFF_CONFIG}" != "x" ]]; then echo "--config ${SQLFLUFF_CONFIG}"; fi) \
```

Shellcheck (0.11.0)

[style] Avoid x-prefix in comparisons as it no longer serves a purpose.

(SC2268)